packages/backend-defaults/src/entrypoints/cache/CacheManager.ts

```diff
@@ -139,12 +139,12 @@ export class CacheManager {
   }
 
   private createRedisStoreFactory(): StoreFactory {
-    const KeyvRedis = require('@keyv/redis');
+    const KeyvRedis = require('@keyv/redis').default;
     let store: typeof KeyvRedis | undefined;
     return (pluginId, defaultTtl) => {
       if (!store) {
         store = new KeyvRedis(this.connection, {
-          useRedisSets: this.useRedisSets,
+          keyPrefixSeparator: ':',
```

Comment:
In v4 the library by default uses `::` as a separator, this should make sure that the keys do not change for default configuration